packages/api/package.json

```diff
@@ -2,11 +2,13 @@
   "name": "api",
   "version": "7.0.0-alpha.3",
   "description": "Magical SDK generation from an OpenAPI definition 🪄",
+  "type": "module",
```

Comment:
`api` is now ESM-only. Codegen'd SDKs will support ESM and CJS but because `api` is a CLI-only tool we don't need to accommodate both module flows.